### PR TITLE
Scoping yo

### PR DESCRIPTION
**Pull request by hl**
Issue #3

### CODE REVIEW

**Comment by zenry** [comment]
Issue #3

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -1,74 +1,55 @@
...
+    redirect_to channel_products_path, notice: 'Product was successfully destroyed.'
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_product
-      @product = Product.find(params[:id])
-    end
+  def set_channel
+    @channel = current_user.channels.find_by_id params[:channel_id]
```
Comment (jeroenvisser101) [inline app/controllers/products_controller.rb:45]: Didn't `.find` work here?

**Comment by jeroenvisser101** [comment]
LGTM

File: app/controllers/products_controller.rb (hunk 1 above)
Comment (hl) [inline app/controllers/products_controller.rb:45]: habit :smile:

Comment (jeroenvisser101) [inline app/controllers/products_controller.rb:45]: haha, it's fine anyways ;)

**Comment by hl** [comment]
Merge it if you like it

**Pull request merged**